proto/substrait/algebra.proto

```diff
@@ -364,6 +364,76 @@ message Rel {
   }
 }
 
+// A base table for writing. The list of string is used to represent namespacing (e.g., mydb.mytable).
+// This assumes shared catalog between systems exchanging a message.
+// it also includes a base schema, and default types
+message NamedTableWrite {
```

Comment:
Could you explain the rationale for the fields in this message? I thought the `DdlRel.query` field already defines the schema except for the names; this schema-and-names pattern is also seen in `RelRoot` adding names to its `Rel input`'s schema. I'm also not sure why the default values are not implicit in the query.